apps/rush/src/taskRunner/ProjectBuildTask.ts

```diff
@@ -244,8 +244,10 @@ export default class ProjectBuildTask implements ITaskDefinition {
     const command: string[] = rawCommand.split(' ');
```

Comment:
>rawCommand.split(' '); [](start = 30, length = 22)

This a broken way to parse a command line.  @nickpape-msft  Could you open a PBI to fix the executeCommand() issue we discussed yesterday?